bindings/cs/Scripts/build.cake

```diff
@@ -0,0 +1,218 @@
+            {
+                BranchName = branch,
+                Checkout = true,
+                IsBare = false,
+                RecurseSubmodules = true
+            });
+
+        EnsureDirectoryExists(libmongocryptRelWithDebInfoDirectory);
+        EnsureDirectoryExists(downloadedMongocryptDirectory);
+        CopyFile(
+            libmongocryptAllDirectory.Combine("windows-test").Combine("bin").CombineWithFilePath("mongocrypt.dll"),
+            downloadedMongocryptDirectory.CombineWithFilePath("mongocrypt.dll"));
+        CopyFile(
+            libmongocryptAllDirectory.Combine("ubuntu1804-64").Combine("nocrypto").Combine("lib").CombineWithFilePath("libmongocrypt.so"),
+            downloadedMongocryptDirectory.CombineWithFilePath("libmongocrypt.so"));
+        CopyFile(
+            libmongocryptAllDirectory.Combine("macos").Combine("nocrypto").Combine("lib").CombineWithFilePath("libmongocrypt.dylib"),
+            downloadedMongocryptDirectory.CombineWithFilePath("libmongocrypt.dylib"));
+        CopyDirectory(downloadedMongocryptDirectory, libmongocryptRelWithDebInfoDirectory);
+    });
+
+Task("Tests")
+    .IsDependentOn("Prepare")
+    .DoesForEach(
+    () => 
+    {
+        var monikersDetails = new List<(string Moniker, string Bitness)>
+        {
+            { ("net452", "x64") },
+            { ("netcoreapp1.1", "x64") },
+            { ("netcoreapp2.1", "x64") },
+            { ("netcoreapp3.0", "x64") },
+            { ("net50", "x64") }
+        };
+        return monikersDetails;
+    },
+    (monikerInfo) =>
+    {
+        Information($"Test running {monikerInfo.Moniker}..");
+        var settings = new DotNetCoreTestSettings
+        {
+            Configuration = configuration,
+            Framework = monikerInfo.Moniker,
+            ArgumentCustomization = args => args.Append($"-- RunConfiguration.TargetPlatform={monikerInfo.Bitness}")
+        };
+        var projectFullPath = libmongocryptTestsSolutionDirectory.CombineWithFilePath("MongoDB.Libmongocrypt.Test.csproj").FullPath;
+        Information(projectFullPath);
+        DotNetCoreTest(
+            projectFullPath,
+            settings
+        );
+    })
+    .DeferOnError();
+    
+Task("CreatePackage")
+    .IsDependentOn("Tests")
+    .Does(() =>
+    {
+        var projectFullPath = libmongocryptSolutionFile.FullPath;
+        var settings = new DotNetCorePackSettings
+        {
+            Configuration = configuration,
+            OutputDirectory = artifactsDirectory
+        };
+        DotNetCorePack(
+            projectFullPath,
+            settings);            
+    });
+    
+Task("NugetPush")
+    .Does(() =>
+    {
+        ThrowIfLocalRelease();
+
+        var nugetApi = GetSettingValue("NugetApiKey", null);
+        var packageFilePath = artifactsDirectory.CombineWithFilePath($"{libmongocryptSolutionFile.GetFilename().ToString()}.{releaseVersion}.nupkg");
+        Information(packageFilePath);
+        NuGetPush(
```

Comment:
I didn't check this, so we can do it during the next release

Reply:
Agreed.